source/Calamari.Common/CalamariFlavourProgramAsync.cs

```diff
@@ -99,6 +101,10 @@ protected async Task<int> Run(string[] args)
         {
             try
             {
+                // adds default max timout to avoid ReDoS attacks
+                // https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.matchtimeout
+                AppDomain.CurrentDomain.SetData("REGEX_DEFAULT_MATCH_TIMEOUT", DefaultRegexMatchTimeout);
+                
                 SecurityProtocols.EnableAllSecurityProtocols();
```

Comment:
It would be nice to follow the convention here and create a static method on a class that describes what is happening. You could then either keep the reasoning comment in that class, or perhaps remove its necessity through naming.